plugins/dell-dock/dell-dock.quirk

```diff
@@ -115,3 +126,20 @@ DellDockUnlockTarget = 10
 DellDockHubVersionLowest = 1.31
 DellDockBlobMajorOffset = 0x400a
 DellDockBlobMinorOffset = 0x4009
+
+# USB4 controller
+[TBT-00d4b071]
+Name = USB4 controller in Dell dock
+Summary = USB4 controller
+Vendor = Dell Inc.
+VendorId = TBT:0x00D4
+ParentGuid = USB\VID_413C&PID_B06E&hub&embedded
+FirmwareSizeMin=0x40000
+FirmwareSizeMax=0x80000
+Flags = skips-restart,require-ac,dual-image
+Icon = thunderbolt
+InstallDuration = 26
+DellDockUnlockTarget = 10
+DellDockHubVersionLowest = 1.31
+DellDockBlobMajorOffset = 0x400a
+DellDockBlobMinorOffset = 0x4009
```

Comment:
I don't think you want these keys - they're only used for I2C based updater, no?

Reply:
yes, i2c-tbt.c isn't needed. I will remove the offset pattern.

Reply:
Done.